lib/sepa.test.js

```diff
@@ -123,6 +124,44 @@ describe('xml generation for transfer documents', () => {
     expect(debtorId).not.toBeUndefined();
     expect(debtorId.textContent).toBe(validDutchCreditorId);
   });
+  
+  test('ctry and address field not null', () => {
+    // GIVEN
+    const doc = validTransferDocument({debtorId: 'FR72ZZZ123456', debtorName: 'debtor-name'});
+    
+    let info = doc._paymentInfo[0];
```

Comment:
Can you extend the functions `validTransferDocument` and `addTransaction` to take the debtor and creditor values. This will make the test a bit more concise but more importantly will avoid using the private fields `_paymentInfo` and `_payments` .